app/routes/index.tsx

```diff
@@ -68,7 +69,7 @@ const AppWrapper = (props) => (
           <Route path="/pages" component={Pages} />
           <Route path="/photos" component={Photos} />
           <Route path="/polls" component={Polls} />
-          <Route path="/quotes" component={Quotes} />
+          <CompatRoute path="/quotes" component={Quotes} />
```

Comment:
Does this need to be CompatRoute? Seems to work fine without

Reply:
You're right, and it is already fixed in the second batch heh